docs/source/configuration.md

```diff
@@ -439,78 +433,77 @@ Parameters used to smooth the shape of the return paths that are displaced in th
+- `pcb_interconnection_method` (Type: `str`, Default: `'spiral_in_out'`)
 
-Interconnection method for PCB: 'spiral_in_out' or 'other'.
+  Interconnection method for PCB: 'spiral_in_out' or 'other'.
 
-- `pcb_spiral_end_shift_factor` (**Type:** `int`, **Default:** `10`)
+- `pcb_spiral_end_shift_factor` (Type: `int`, Default: `10`)
 
-Factor (as a percent) to shift the open ends of the spirals in order to avoid overlaps.
+  Factor (as a percent) to shift the open ends of the spirals in order to avoid overlaps.
 
 ### Generate 3D Wire Path
 
 The application can optionally generate a 3D `.stl` trace by sweeping out a conductor profile along the computed wire path.
 
-- `skip_sweep` (**Type:** `bool`, **Default:** `False`)
+- `skip_sweep` (Type: `bool`, Default: `False`)
 
-Flag to skip the generation of a volumetric (3D) coil body.
+  If True, skips the generation of a volumetric (3D) coil body.
 
-The calculated 3D surface is stored in `layout_surface_mesh` property.
+  The calculated 3D surface is stored in the `layout_surface_mesh` property.
 
-- `cross_sectional_points` (**Type:** `list of float`, **Default:** `[0, 0]`)
+- `cross_sectional_points` (Type: `list of float`, Default: `[0, 0]`)
 
-This parameter describes the 2D profile of the conductor surface. 
+  This parameter describes the 2D profile of the conductor surface. 
 
-The default values of `[0,0]` instructs the application to generate a 10-sided circular profile with a radius
-specified by the `conductor_thickness` parameter.
+  The default of `[0,0]` instructs `pyCoilGen` to generate a 10-sided circular profile with a radius
+  specified by the `conductor_thickness` parameter.
 
-A custom shape defined by specifying the x/y co-ordinates in metres in a 2xm array of the form `[[x0, x1, x2, x3, ...], [y0, y1, y2, y3, ...]]`.
+  A custom shape defined by specifying the x/y co-ordinates in metres in a 2xm array of the form `[[x0, x1, x2, x3, ...], [y0, y1, y2, y3, ...]]`.
 
 
-- `save_stl_flag` (**Type:** `bool`, **Default:** `True`)
+- `save_stl_flag` (Type: `bool`, Default: `True`)
 
-Flag to save the swept conductor profile to an .stl file.
+  If True, saves the swept conductor profile to an `.stl` file.
 
-If `skip_sweep` is False and `save_stl_flag` is True, the generated result is saved in the output_directory, with a name corresponding 
-to `{project_name}_surface_part{part_ind}_{field_shape_function}.stl`, where `part_ind` is the zero-based index of the winding coil mesh parts.
+  If `skip_sweep` is False and `save_stl_flag` is True, the generated result is saved in the output_directory, with a name corresponding 
+  to `{project_name}_surface_part{part_ind}_{field_shape_function}.stl`, where `part_ind` is the zero-based index of the winding coil mesh parts.
 
-The `field_shape_function` is stripped of any `*`, `^`, and `,` symbols.
+  The `field_shape_function` is stripped of any `*`, `^`, and `,` symbols.
 
 
 ## Evaluate Results
 
```

Comment:
Add a one sentence intro to what this section contains - as the other sections have.

Reply:
Addressed in commit 3780bc6.